src/AddOn.Episerver.Settings/Core/SettingsService.cs

```diff
@@ -129,12 +146,34 @@ public class SettingsService : ISettingsService
         /// </summary>
         /// <value>The global settings root.</value>
         public ContentReference GlobalSettingsRoot { get; set; }
-
+        
         /// <summary>
         /// Gets or sets the settings root.
         /// </summary>
-        /// <value>The settings root.</value>
+        /// <value>The global settings root.</value>
```

Comment:
Are we keeping this for backwards compability. Seems to be confusing to have two properties that both describes that they return the global settings root.